app/events/base_event.py

```diff
@@ -226,3 +226,15 @@ async def audience_overlap_success_event_handler(cls, data: Dict):
     @classmethod
     async def audience_overlap_failure_event_handler(cls, data: Dict):
         pass
+
+    @classmethod
+    async def professional_contents_fetch_request_event_handler(cls, data: Dict):
```

Comment:
request is not an event type
```suggestion
    async def professional_contents_fetch_request_handler(cls, data: Dict):
```